misc/review

```diff
@@ -126,11 +126,29 @@ inspect_cmd_skip ()
     echo "SKIPPING $1" 1>&2
 }
 
-inspect_cmd_show_diff ()
+inspect_cmd_show_genenric ()
```

Comment:
Thanks. I will decide to write a script for checking spells and integrate it to "make check".